boards/arm/stm32vl_disco/doc/index.rst

```diff
@@ -0,0 +1,191 @@
+Line" STM32F100x SoC series is showcased. Like other Discovery board, an
+integrated ST-LINK debugger and programmer is included (V1), but the only
+included I/O devices are two user LEDs and one user button.
+
+.. image:: img/stm32vl_disco.jpg
+     :width: 552px
+     :height: 1024px
+     :align: center
+     :alt: STM32VLDISCOVERY
+
+More information about the board can be found at the `STM32VLDISCOVERY website`_.
+
+Hardware
+********
+
+The STM32 Discovery board features:
+
+- On-board ST-LINK/V1 with selection mode switch to use the kit as a standalone
+  ST-LINK/V1 (with SWD connector for programming and debugging)
+- Board power supply: through USB bus or from an external 5 V supply voltage
+- External application power supply: 3 V and 5 V
+- Four LEDs:
+
+    - LD1 (red) for 3.3 V power on
+    - LD2 (red/green) for USB communication
+    - LD3 (green) for PC9 output
+    - LD4 (blue) for PC8 output
+- Two push buttons (user and reset)
+- Extension header for all LQFP64 I/Os for quick connection to prototyping board
+  and easy probing
+
+More information about the STM32F100x can be found in the
+`STM32F100x reference manual`_ and the `STM32F100x data sheet`_.
+
+Supported Features
+==================
+
+The Zephyr stm32vl_disco board configuration supports the following hardware features:
+
+.. list-table:: Supported hardware
+   :header-rows: 1
+
+   * - Interface
+     - Controller
+     - Driver/component
+   * - NVIC
+     - on-chip
+     - nested vector interrupt controller
+   * - UART
+     - on-chip
+     - serial port-polling
+       serial port-interrupt
+   * - PINMUX
+     - on-chip
+     - pinmux
+   * - GPIO
+     - on-chip
+     - gpio
+   * - CLOCK
+     - on-chip
+     - reset and clock control
+   * - FLASH
+     - on-chip
+     - flash memory
+   * - WATCHDOG
+     - on-chip
+     - window watchdog
+   * - I2C
+     - on-chip
+     - i2c
+   * - SPI
+     - on-chip
+     - spi
+   * - ADC
+     - on-chip
+     - adc
+
+Other hardware features are not yet supported in this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/stm32vl_disco/stm32vl_disco_defconfig``
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down), or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+.. rst-class:: rst-columns
+
+- UART_1_TX : PA9
+- UART_1_RX : PA10
+- UART_2_TX : PA2
+- UART_2_RX : PA3
+- UART_2_TX : PB10
+- UART_2_RX : PB11
+- SPI1_NSS : PA4
+- SPI1_SCK : PA5
+- SPI1_MISO : PA6
+- SPI1_MOSI : PA7
+- SPI2_NSS : PB12
+- SPI2_SCK : PB13
+- SPI2_MISO : PB14
+- SPI2_MOSI : PB15
+- I2C1_SCL : PB6
+- I2C1_SDA : PB7
+- I2C2_SCL : PB10
+- I2C2_SDA : PB11
+- ADC1_IN0 : PA0
```

Comment:
Remove this line

Reply:
Nice catch. Will do that

Reply:
I'm going for broke and adding the Xe dtsi, copying the method from 103Xe and including `dts/arm/st/f1/stm32f100Xb.dtsi` from `dts/arm/st/f1/stm32f100Xe.dtsi` .